wangle/acceptor/TransportInfo.h

```diff
@@ -104,18 +104,17 @@ struct TransportInfo {
    */
   int64_t ssthresh{-1};
 
-#if defined(__linux__) || defined(__FreeBSD__)
+#ifdef __APPLE__
+  typedef tcp_connection_info tcp_info;
+  static const int TCP_INFO = TCP_CONNECTION_INFO;
```

Comment:
`TCP_INFO` specifically can be a `#define` in the `.cpp` since that is the only place it is used by wangle.

Reply:
Okay, I made the change.